markdown/usage/troubleshooting.md

```diff
@@ -24,6 +28,81 @@ subtitle: How to troubleshoot common mistakes and issues
 - [Using a local version of iGenomes](#using-a-local-version-of-igenomes)
 - [Extra resources and getting help](#extra-resources-and-getting-help)
 
+
+## Troubleshooting basics
+
+These are the recommended steps for troubleshooting a pipeline. 
+
+### Start small
+
+Before using the pipeline with your data, make sure to run a test using: 
+
+```bash
+nextflow run nf-core/<pipeline_name> -profile test,docker
+```
+
+If Docker is not installed, you can replace `docker` with `singularity` or `conda`, see
+the [Getting Started](https://nf-co.re/usage/introduction) tutorial for further information. If a test fails, it might indicate that 
+there is an issue with installation or configuration rather than a pipeline error.
+
+You might also want to check the following:
+
+1. Nextflow is up to date. Use `nextflow self-update` to update a typical installation or `conda update nextflow` for a Bioconda installation.
+2. There is enough disk space, this will avoid running out of space while you are running the pipeline. 
+3. Docker daemon is running (if you are using Docker to manage dependencies).
+
+
+### Categorize the type of error
+
+For this step you try to identify when the error occurs:
+
+1. Before the first process.
+2. During the first process. 
+3. During run.
```

Comment:
```suggestion
3. During run: for errors that occur while a pipeline is running or generating outputs it might be helpful to check log files as explained below.
```